### PR TITLE
Fix broken links, remove absolute links

### PR DESCRIPTION
**Pull request by kolcun**
updated broken link to homekit docs

### CODE REVIEW

File: configuration/items.md [hunk 1]
```diff
@@ -524,7 +524,7 @@ Tagging is a new feature and only a few I/O add-ons have implemented it.
 The easiest way to determine if tags have been implemented in a specific add-on is to see if the add-on documentation explicitly discusses their usage.
 Tags will be ignored if no Items in the openHAB installation support it.
 
-See the [Hue Emulation]({{base}}/addons/io/hueemulation/readme.html) or [HomeKit Add-on](http://docs.openhab.org/addons/io/homekit/readme.html) documentation for more details.
+See the [Hue Emulation]({{base}}/addons/io/hueemulation/readme.html) or [HomeKit Add-on](http://docs.openhab.org/addons/ios/homekit/readme.html) documentation for more details.
```
Comment (Confectrician) [inline configuration/items.md]: Could you change the Hue Emulation link accordingly?
It has to be changed to `ios` too and with that we don't need another Pull Request.

**Comment by kolcun** [comment]
Updated - good catch

**Comment by Confectrician** [comment]
Looks good. 👍 
Good catch with the base-template.

---
### Some general "First-time contributor" info

Since this is a one-line change you don't need to **sign-off** your work for _this_ Pull Request.
But for possible improvements in the future i want to point you to our [Contribution Guideline](https://github.com/openhab/openhab-docs/blob/gh-pages/CONTRIBUTING.md)

---
Again thanks for your contribution.
I will approve and merge this later.

**Comment by Confectrician** [comment]
@ThomDietrich 

As i already wrote:
> Since this is a one-line change you don't need to sign-off your work for this Pull Request.

The info was meant as a generic hint to the contribution guidelines 😃

**Comment by ThomDietrich** [comment]
😅 sorry. Didn't read past the bold "sing-off"

**Pull request merged**